data/external.yaml

```diff
@@ -14,26 +14,26 @@ PhotoSwipe:
       type: script
       defer: true
 
-    - src: https://cdn.jsdelivr.net/npm/photoswipe@4.1.3/dist/default-skin/default-skin.css
-      integrity: sha256-c0uckgykQ9v5k+IqViZOZKc47Jn7KQil4/MP3ySA3F8=
+    - src: https://cdn.jsdelivr.net/npm/photoswipe@4.1.3/dist/default-skin/default-skin.min.css
+      integrity: sha256-KJg1buMpDOS7Vvj+W5e2Cr3geBzBWZ0L9ZGME7j35U4=
       type: style
 
-    - src: https://cdn.jsdelivr.net/npm/photoswipe@4.1.3/dist/photoswipe.css
-      integrity: sha256-SBLU4vv6CA6lHsZ1XyTdhyjJxCjPif/TRkjnsyGAGnE=
+    - src: https://cdn.jsdelivr.net/npm/photoswipe@4.1.3/dist/photoswipe.min.css
+      integrity: sha256-Zfb0kT83RyzMLwWKIYyl4XPjTvVNsNRKDxelJp2L9VE=
```

Comment:
`min.css` is generated dynamically by JSDelivr, and it's said not to use SRI with it ("Do NOT use SRI with dynamically generated files! More information: https://www.jsdelivr.com/using-sri-with-dynamic-files" if you visit that file).

Maybe we can remove the SRI support (remove integrity field) from these two files.